enterprise/workspaceapps_test.go

```diff
@@ -59,6 +67,13 @@ func TestWorkspaceApps(t *testing.T) {
 			},
 		})
 
+		_ = dbgen.CryptoKey(t, db, database.CryptoKey{
+			Feature: database.CryptoKeyFeatureWorkspaceAppsToken,
+		})
+		_ = dbgen.CryptoKey(t, db, database.CryptoKey{
+			Feature: database.CryptoKeyFeatureWorkspaceAppsAPIKey,
+		})
```

Comment:
This strikes me as an argument for making the key rotator part of the `API`, or at least part of `coderdtest` / `coderdenttest`.  We treat these as being a "fully functioning" coderd, and without crypto keys, certain parts are not functional.

Other test writers are going to get bit with not having the crypto keys and will be confused.

Reply:
Yeah I actually had it this way and moved it out but I think I regret that. I think we move it into the API.